src/data/screenshots.json

```diff
@@ -10,7 +10,7 @@
         "content": {
             "textblock": [
                 "Browse through the screenshots to get an overview of the features that Corona-Warn-App offers in the current release (Version 2.7 available since 16.07.2021). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
```

Comment:
The date is wrong! It should be 16.08.2021 (August, not July).

Reply:
True, thanks for the hint! Fixed in [`95ea03d` (#1635)](https://github.com/corona-warn-app/cwa-website/pull/1635/commits/95ea03d5976abc12cbb5dc8cde1e059a629819dd)